conf/app.routes

```diff
@@ -111,13 +111,15 @@ GET         /schools                                                     control
 
 POST        /assessor/save/:userId                                       controllers.AssessorController.saveAssessor(userId: String)
 GET         /assessor/find/:userId                                       controllers.AssessorController.findAssessor(userId: String)
-POST        /assessor/availability/save/:userId                          controllers.AssessorController.addAvailability(userId: String)
-GET         /assessor/availability/find/:userId                          controllers.AssessorController.findAvailability(userId: String)
+
+POST        /assessor-availability/save/:userId                          controllers.AssessorController.addAvailability(userId: String)
```

Comment:
I think we have change those end points to be /assessor/availability because availability is part of assessors, you will get a conflict probably to resolve.